bindata/v4.1.0/kube-apiserver/pod.yaml

```diff
@@ -43,6 +43,15 @@ spec:
     command: ["/bin/bash", "-ec"]
     args:
         - |
+          echo -n "Acquiring exclusive lock /var/log/kube-apiserver/.lock"
+          exec {lock_fd}>/var/log/kube-apiserver/.lock || exit 1
+          flock -n "${lock_fd}" || {
+            echo "$(date -Iseconds -u) kubelet did not terminate old kube-apiserver before new one" >> /var/log/kube-apiserver/lock.log
+            echo " WARNING: kubelet did not terminate old kube-apiserver before new one." >&2
+            flock -s -n "${lock_fd}"
```

Comment:
@smarterclayton please see here. We made it not fail anymore, but just switch to shared lock mode. So the next container will try the exclusive lock again and will warn again. This way we get all the debugging value, but don't fail starting the pod.

Reply:
Ok, thanks